src/pytest_benchmark/utils.py

```diff
@@ -114,11 +114,27 @@ def get_project_name():
     else:
         return os.path.basename(os.getcwd())
 
+def get_branch_info():
+    if os.path.exists('.git'):
+        try:
+            branch = check_output('git symbolic-ref --short -q HEAD'.split())
+            return branch.strip()
+        except subprocess.CalledProcessError:
+            return '(detached head)'
```

Comment:
Shouldn't the others be marked somehow specially when unknown? Maybe `"(unknown)"` or `"(error: something)"`